browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2213,7 +2213,6 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			$(control.container).unbind('click');
 			$(control.container).click(function () {
 				$(control.container).w2menu({
-					items: builder._menus[menuId],
```

Comment:
This should stay here to show menu for menubartoolitem.
I assume this is added by mistake.

Reply:
yup, my mistake